.github/workflows/publish-command.yml

```diff
@@ -80,6 +80,7 @@ jobs:
           STRIPE_INTEGRATION_TEST_CREDS: ${{ secrets.STRIPE_INTEGRATION_TEST_CREDS }}
           TEMPO_INTEGRATION_TEST_CREDS: ${{ secrets.TEMPO_INTEGRATION_TEST_CREDS }}
           TWILIO_TEST_CREDS: ${{ secrets.TWILIO_TEST_CREDS }}
+          ZENDESK_TALK_SECRETS_CREDS: ${{ secrets.ZENDESK_TALK_SECRETS_CREDS }}
```

Comment:
are you sure this is the right name? IIRC it should be `ZENDESK_TALK_TEST_CREDS`?